app/jobs/appointment_notification/worker.rb

```diff
@@ -4,27 +4,28 @@ class AppointmentNotification::Worker
 
   sidekiq_options queue: :high
 
-  DEFAULT_LOCALE = :en
-
   def metrics
     @metrics ||= Metrics.with_object(self)
   end
 
-  def perform(appointment_reminder_id, communication_type)
+  def perform(appointment_reminder_id)
     metrics.increment("attempts")
     unless Flipper.enabled?(:appointment_reminders)
       metrics.increment("skipped.feature_disabled")
       return
     end
     reminder = AppointmentReminder.includes(:appointment, :patient).find(appointment_reminder_id)
```

Comment:
Can you chain the `find` first and then includes? It reads better as the `find` is the more important part of the lookup here.

Reply:
AFAIK that's just not how the syntax works. https://stackoverflow.com/questions/3943554/how-to-do-find-with-includes-in-rails-3

Reply:
👍

Reply:
Got it.